main.py

```diff
@@ -41,6 +41,8 @@
 
 
 SETTINGS = Settings()
+SETTINGS.load_from_environment()
```

Comment:
putting these things in global breaks the tests. sorry, I wanted to get around to refactor this a bit more but in general, we need to avoid using globals in the scripts as it makes re-use very difficult.

maybe even load these inside `slack_notify` instead?

Reply:
True, let's load them in `slack_notify()` for now. Thanks.

Reply:
Hmm, actually I went to change this but it doesn't seem right to do that either.

I'll just keep running the script manually every morning until I get time to refactor our the global settings variable.